### PR TITLE
Contribute section: new ideas for pythonizations

### PR DESCRIPTION
**Pull request by etejedor**

### CODE REVIEW

File: _ideas/2020-06-26-add-a-root-pythonization.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+summary: Add pythonizations for ROOT classes, which make it easier to use ROOT from Python.
+layout: single
+author: Enric Tejedor
+state:
+
+sidebar:
+  nav: "contribute"
+---
+
+ROOT's Python bindings, called [PyROOT](https://root.cern/manual/python), allow to access all the ROOT C++ functionality
```
Comment (couet) [inline _ideas/2020-06-26-add-a-root-pythonization.md:12]: Absolute link like that will not work. 
`[PyROOT](https://root.cern/manual/python)` should be something like:
`[PyROOT]({{'manual/python' | relative_url}})`

**Comment by eguiraud** [comment]
If this is ok for @couet , we can merge, the failure in HTML proofer is unrelated and caused by a push by @Axel-Naumann I think

**Comment by couet** [comment]
fine for me

**Comment by Axel-Naumann** [comment]
And my breakage is now unbroken!

**Pull request merged**